packages/shared/components/inputs/Input.svelte

```diff
@@ -19,15 +19,23 @@
     export let isFocused = false
     export let maxDecimals = undefined
     export let disableContextMenu = false
+    export let locale = undefined
+    export let capsLockWarning = false
 
     let inputElement
     let allDecimalSeparators = getAllDecimalSeparators()
     let decimalSeparator = getDecimalSeparator()
+    let capsLockOn = false
+    let hasFocus = false
 
     const handleInput = (e) => {
         value = e.target.value
     }
 
+    const onKeyDown = (e) => {
```

Comment:
I'm not sure if this is specific to macOS, but for some reason `keydown` is not fired when caps lock is turned off (instead I think `keyup` is fired). As a result, the caps lock warning shows until the next character is pressed, which might be a minor UX issue (not a huge deal though)

Reply:
I have added the handler on keyup as well, both can get the modified state without a problem, so that should catch both cases.